.github/workflows/frontend.yml

```diff
@@ -0,0 +1,32 @@
+name: Frontend npm Tests
+
+on:
+  pull_request:
+    paths:
+      - "plugins/management-console/**"
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: 11
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      # Enable gradle cache: https://github.com/actions/cache/blob/master/examples.md#java---gradle
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: run npm build
+        run: |
+          cd plugins/management-console
+          ./gradlew bCA
```

Comment:
This is part of gradle build already, we don't need a separate workflow

Reply:
How do you run this without setting up npm?

Reply:
gradle node plugin download the node and npm